st-app-frontend/src/app/components/Footer/Feedback.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { ThumbsUp } from "lucide-react";
+import { Trophy } from "lucide-react";
+import { Heart } from "lucide-react";
+
+import { Menu } from "./Menu";
+
+const publicRoutes = [
+  {
+    icon: (
+      <ThumbsUp
+        color="blue"
+        size={16}

```

Comment:
(not critical)
I would prefer if we define the size of the icons as a const and then use in the list.